src/persist/src/indexed/runtime.rs

```diff
@@ -102,40 +104,27 @@ where
     let indexed = Indexed::new(log, blob, maintainer, metrics.clone())?;
     let mut runtime = RuntimeImpl::new(config.clone(), indexed, rx, metrics.clone());
     let id = RuntimeId::new();
-    let runtime_pool = pool.clone();
-    let impl_handle = thread::Builder::new()
```

Comment:
lets keep the persist runtime as a dedicated thread for now? I realize that means two distinct types of joinhandle in `RuntimeHandles` but I'm not confident enough in my understanding of the tokio runtime to say if the persistence runtime should be moved to that vs its own thread. for the ticker task its much more clear that its not doing anything computationally intensive